configuration/defaults.go

```diff
@@ -65,5 +65,7 @@ func SetDefaults(settings *viper.Viper) {
 	settings.BindEnv("directories.User", "ARDUINO_SKETCHBOOK_DIR")
 	settings.BindEnv("directories.Downloads", "ARDUINO_DOWNLOADS_DIR")
 	settings.BindEnv("directories.Data", "ARDUINO_DATA_DIR")
+	settings.BindEnv("directories.builtin.tools", "ARDUINO_BUILTIN_TOOLS_DIR")
+	settings.BindEnv("directories.builtin.libraires", "ARDUINO_BUILTIN_LIBRARIES_DIR")
```

Comment:
```suggestion
	settings.BindEnv("directories.builtin.libraries", "ARDUINO_BUILTIN_LIBRARIES_DIR")
```
Fix typo in key name.

But more importantly, are these environment variables even needed? From the comment in this file, it seems the purpose is for backwards compatibility with some previous version that apparently used them. But that doesn't seem to be the case with these two environment variables from what I can tell.

If they are not needed for backwards compatibility, I recommend removing them. The documented and standard auto-generated environment variables (e.g., `ARDUINO_DIRECTORIES_BUILTIN_LIBRARIES`) can be used instead.

Reply:
Right, better remove them.